scripts/taxonomy/GetTaxonomyTable.pl

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env perl
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2019] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+use Getopt::Long;
+use Cwd qw(abs_path);
+use File::Spec::Functions qw(catfile);
+use Bio::EnsEMBL::Compara::DBSQL::DBAdaptor;
+
+$| = 1;
```

Comment:
This is only useful when the script is printing its ouput in a "stream" manner. Here you write everything at once at the end, so there won't be any visible differences